odxtools/diagdatadictionaryspec.py

```diff
@@ -46,9 +46,7 @@ class DiagDataDictionarySpec:
     @staticmethod
     def from_et(et_element: ElementTree.Element,
                 context: OdxDocContext) -> "DiagDataDictionarySpec":
-        admin_data = None
-        if (admin_data_elem := et_element.find("ADMIN-DATA")) is not None:
-            admin_data = AdminData.from_et(admin_data_elem, context)
+        admin_data = AdminData.from_et(et_element.find("ADMIN-DATA"), context)
```

Comment:
Why?

Reply:
`AdminData.from_et()` accepts `None` for the `et_element` and returns `None` in this case. This simplifies the many places where `AdminData` is required. (Same for SDGs, BTW.)